src/MspFcComms.cpp

```diff
@@ -78,12 +78,14 @@ namespace FcComms
     {
         uint16_t raw_values[18];
         getRawRC(raw_values);
-        char RC_info[100];
-        int j = 0;
-        for (int i = 0 ; i < 18 ; i++) {
-            j+=sprintf(&RC_info[j], ", %d", raw_values[i]);
-        } //This is Levi's for loop
-        ROS_INFO(RC_info);
+
+        std::string RC_info;
```

Comment:
Why are we using dynamic memory on the flight controller?  Also what's wrong with `snprintf`?